File: 3553-check-if-two-chessboard-squares-have-the-same-color/check-if-two-chessboard-squares-have-the-same-color.py
class Solution:
    def checkTwoChessboards(self, coordinate1: str, coordinate2: str) -> bool:
        def get_color(coordinate):
            column = ord(coordinate[0]) - ord('a') + 1
            row = int(coordinate[1])
            return (column + row) % 2
        return get_color(coordinate1) == get_color(coordinate2)

File: 3553-check-if-two-chessboard-squares-have-the-same-color/check-if-two-chessboard-squares-have-the-same-color.js
/**
 * @param {string} coordinate1
 * @param {string} coordinate2
 * @return {boolean}
 */
var checkTwoChessboards = function(coordinate1, coordinate2) {
    const sumCoord1 = (coordinate1[0].charCodeAt(0) - 96) + Number(coordinate1[1]);
    const sumCoord2 = (coordinate2[0].charCodeAt(0) - 96) + Number(coordinate2[1]);

    if (sumCoord1 % 2 === sumCoord2 % 2) {
        return true;
    }
    return false;
};